src/utils.ts

```diff
@@ -0,0 +1,27 @@
+import { Contract } from '@ethersproject/contracts';
+import { StaticJsonRpcProvider } from '@ethersproject/providers';
+import { hexValue } from '@ethersproject/bytes';
+import fetch from 'node-fetch';
+
+async function call(provider, abi, call, options = {}) {
+  const contract = new Contract(call[0], abi, provider);
+  try {
+    const params = call[2] || [];
+    return await contract[call[1]](...params, options);
+  } catch (e) {
+    return Promise.reject(e);
+  }
+}
+
+export async function getSafeVersion(safe) {
+  const networks = await fetch(
+    'https://raw.githubusercontent.com/snapshot-labs/snapshot.js/master/src/networks.json'
+  ).then(res => res.json());
+  const rpc = networks['1'].rpc[0];
+  const connectionInfo = typeof rpc === 'object' ? { ...rpc, timeout: 25000 } : { url: rpc, timeout: 25000 };
+  const provider = new StaticJsonRpcProvider(connectionInfo);
```

Comment:
I would just do like this here
```suggestion
  const provider = new StaticJsonRpcProvider('https://rpc.ankr.com/eth');
```
Similar than within the stamp server to keep it simple.

Reply:
What if this node fails/has a downtime? 🤔 we may have to edit all these manually right?

Reply:
Yes, but this is already the case with others servers. We need a solution but we dont need to address this on that repo.

Reply:
How about a proxy node? so we will have only one place to change the node (instead of changing node in both relayer, stamp, and others)

Reply:
We already using @snapshot-labs/snapshot.js in this repo, didn't notice that 😝